### ISSUE TITLE
jenkins/jenkinsfile-runner Docker Image can't evaluate Jenkins version

### ISSUE THREAD

**Issue by novettaberin** [jenkinsci/jenkinsfile-runner#423]
The Plugin Installation Manager Plugin that is part of the Docker image first attempts to check the version of the Jenkins war file so that it can find the latest **compatible** dependency for Jenkins Core and the plugins you are specifying.

The net result is that the Plugin Installation Manager Plugin assumes that the version of Jenkins is the most recent, which is not the case for the docker image.  The problem that makes is that when I try to create my own Image with the plugins I need installed I get a problem with the `workflow-basic-steps` plugin.  The install manager errors out saying that I need version 2.23 of that plugin, which is incompatible with the version of Jenkins in the `jenkins/jenkinsfile-runner` Docker Image.

Please fix the issue preventing that plugin from detecting the version of Jenkins in the docker image.  Also, if possible, specify what version that should be.

**Comment by oleg-nenashev** [comment]
Yes, this is a legitimate defect. I was about fixing it in #324 by replacing the plugin manager by an embedded version, but I have not done it yet. I will see whether I could add an alternate solution

**Comment by oleg-nenashev** [comment]
https://github.com/jenkinsci/plugin-installation-manager-tool/pull/232 for the enhancement on the Plugin Installation Manager side that makes it possible

**Comment by oleg-nenashev** [comment]
@bericoberin  I added the new Plugin Installation Manager version in the `latest` images (part of #449 ).
Now you can use the version parameter to ensure the versions are considered.
It would be great to automate detecting a proper version, but it requires a few extra patches.
